src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,56 +1,125 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+interface Props {
+  onAdd: (movieToAdd: Movie) => void;
+}
+
+export const NewMovie: React.FC <Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+
+  const [newTitle, setNewTitle] = useState('');
+  const [newDescription, setNewDescription] = useState('');
+  const [newImgUrl, setNewImgUrl] = useState('');
+  const [newImdbUrl, setNewImdbUrl] = useState('');
+  const [newImdbId, setNewImdbId] = useState('');
+
+  const clearForm = () => {
+    setNewTitle('');
+    setNewDescription('');
+    setNewImgUrl('');
+    setNewImdbUrl('');
+    setNewImdbId('');
+  };
+
+  const validateTextField = (textField: string): string => textField.trim();
```

Comment:
As I understand it trims the string so maybe 
```suggestion
  const trimTextField = (textField: string): string => textField.trim();
```

Reply:
Done.
Thank you!